docs/_guides/codewind-crw-quick-guide.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: guide
+summary_title: "Codewind in CodeReady Workspaces"
+title: "Getting Started with Codewind in CodeReady Workspaces"
+categories: guides 
+guide_description: "Use Codewind in CodeReady Workspaces to build high quality cloud native applications."
+permalink: codewind-crw-quick-guide.html
+duration: 10 minutes
+tags: Codewind, CodeReady Workspaces, Openshift, Kubernetes, containers, microservice
+--- 
+
+## Objectives
+* Install CodeReady Workspaces and Codewind
+* Develop a simple microservice that uses Eclipse Codewind on CodeReady Workspaces
+
+## Overview
+Use Eclipse Codewind to develop microservice applications from application stacks in an integrated developer environment (IDE). CodeReady Workspaces provides a containerized IDE for cloud native application development on an OpenShift cluster. 
+
+## Developing with CodeReady Workspaces 
+CodeReady Workspaces uses Kubernetes and containers to provide a preconfigured environment. Use CodeReady Workspaces to create, build, and test your code in OpenShift containers but feel like you are working on an IDE on your local machine.  
+
+### Prerequisite
+CodeReady Workspaces require at least two 5Gi ReadWriteOnce (RWO) persistent volumes on the cluster to install and a 5Gi RWO volume for each created workspace.
+
+Each Codewind workspace also requires at least on 5Gi ReadWriteMany (RWX) persistent volume.
+
+Before you can use Codewind with CodeReady Workspaces, apply the Codewind cluster role with `oc apply -f`. 
+
+### Installing CodeReady Workspaces
+1\. Subscribe to the CodeReady Workspaces OperatorHub then install CodeReady Workspaces:
+
+![Subscribe to OperatorHub](./images/codereadyworkspaces-images/CRW-OperatorHub.png){:width="1684"}. 
+
+![Install CodeReady Workspaces](./images/codereadyworkspaces-images/CRW-install.png){:width="1626"}. 
+
+  * If you did not set up an auth provider, set `openShiftoAuth` to `false`. 
+  * To enable HTTPS, set `tlsSupport` to `true`.
+  * If you use a self-signed certificate, set `selfSignedCert` to `true`. 
+
+2\. Use the following yaml:  
+```yaml
+apiVersion: org.eclipse.che/v1
```

Comment:
This yaml needs to be updated to:
```
apiVersion: org.eclipse.che/v1
kind: CheCluster
metadata:
  name: codeready-workspaces
spec:
  server:
    cheImageTag: ''
    cheFlavor: codeready
    devfileRegistryImage: ''
    pluginRegistryImage: ''
    tlsSupport: true
    selfSignedCert: true
    cheWorkspaceClusterRole: 'eclipse-codewind'
    customCheProperties:
      CHE_INFRA_KUBERNETES_WORKSPACE__START__TIMEOUT__MIN: "15"
      CHE_LIMITS_WORKSPACE_IDLE_TIMEOUT: "0"
      CHE_WORKSPACE_PLUGIN__BROKER_WAIT__TIMEOUT__MIN: "15"
  database:
    externalDb: false
    chePostgresHostName: ''
    chePostgresPort: ''
    chePostgresUser: ''
    chePostgresPassword: ''
    chePostgresDb: ''
  auth:
    openShiftoAuth: false
    identityProviderImage: ''
    externalIdentityProvider: false
    identityProviderURL: ''
    identityProviderRealm: ''
    identityProviderClientId: ''
  storage:
    pvcStrategy: per-workspace
    pvcClaimSize: 1Gi
    preCreateSubPaths: true
```

Reply:
On it. Will have the new commit up soon.